### PR TITLE
DOCS-1822-Add-Helm-mcm

### PR DESCRIPTION
**Pull request by bmckercher123**
RN 

Product Version(s):
CE 3.19

Issue:
https://tigera.atlassian.net/browse/DOCS-1822

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Review by dimitri-nicolo — APPROVED** [review]
lgtm

File: calico-enterprise/release-notes/index.mdx [hunk 1]
```diff
@@ -14,11 +14,11 @@ Added support for VXLAN overlay networking to federated endpoint indentity and s
 
 For more information, see [Configure federated endpoint identity and services](../multicluster/federation/kubeconfig).
 
-### Feature two
+### Helm install and upgrade for multi-cluster management
 
-Here's what you need to know about this feature.
+Adds Helm install and upgrade for {{prodname}} multi-cluster management. As a de facto standard for deploying software in Kubernetes, Helm provides deployment automation in tools like Gitlab.
```
Comment (ctauchen) [inline calico-enterprise/release-notes/index.mdx]: The second sentence reads a bit sales-y. Perhaps something like:

We've added the option to use Helm to install and upgrade {{prodname}} in multi-cluster settings.
With Helm, your DevOps teams can automate these complex deployments more easily.

File: calico-enterprise/release-notes/index.mdx [hunk 2]
```diff
@@ -14,11 +14,11 @@ Added support for VXLAN overlay networking to federated endpoint indentity and s
 
 For more information, see [Configure federated endpoint identity and services](../multicluster/federation/kubeconfig).
 
-### Feature two
+### Helm install and upgrade for multi-cluster management
```
Comment (ctauchen) [inline calico-enterprise/release-notes/index.mdx]:
```suggestion
### Helm installation and upgrade for multi-cluster deployments
```
A suggestion, take or leave

**Review by ctauchen — COMMENTED** [review]
Some suggestions, otherwise LGTM

File: calico-enterprise/release-notes/index.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/release-notes/index.mdx]: Fixed.

File: calico-enterprise/release-notes/index.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/release-notes/index.mdx]: Took part of it.

**Pull request merged**